TS03-Subset-Data-and-No-Data-Values.Rmd

```diff
@@ -165,7 +162,7 @@ qplot (datetime, prec,
 ## Missing Data - Dealing with data gaps
 
 ###Checking for No Data Values
-In the <a href="XXX" target="_blank"> Understand The Data</a>
+In the  [{{site.baseurl}} / LINKHERE](Lesson 01 - )
```

Comment:
If the link is EXTERNAL, please use HTML <`a href=`>. you can't tell it to open in a new window using markdown. If it is OK for the link to open in the same window (it's another one of our lessons maybe?) use markdown `()[]` There are some limitations to markdown unfortunately.